requests/adapters.py

```diff
@@ -466,11 +466,9 @@ def send(self, request, stream=False, timeout=None, verify=True, cert=None, prox
 
                     # Receive the response from the server
                     try:
-                        # For Python 2.7+ versions, use buffering of HTTP
-                        # responses
                         r = low_conn.getresponse(buffering=True)
                     except TypeError:
-                        # For compatibility with Python 2.6 versions and back
+                        # For compatibility with Python 3.6 versions and up
```

Comment:
Can't this section be changed to remove the `try-except`?

Reply:
What do you suggest?

`r = low_conn.getresponse(buffering=True)` gives `TypeError: getresponse() got an unexpected keyword argument 'buffering'` on Python 3.6.

Reply:
Hm, TIL. Well I guess just leave it as is then. I'll rescind my review.